src/components/current-task/TimerConfig.jsx

```diff
@@ -59,30 +69,52 @@ function TimerConfig(props) {
   const { setTimerConfigValues } = props;
 
   const [isChecked, setIsChecked] = useContext(FullscreenContext);
+  const [isInvalidWorkTime, setIsInvalidWorkTime] = useState(false);
```

Comment:
Hooks should be declared at the start of the component

Reply:
This case was kind of weird, in Timer config they seem to define two components Timerconfig (where the hooks are located at the top of in this case) and a separate IntervalInput component above it, hence these hooks aren't located at the top of the file.

Reply:
Ah, my mistake. I must've missed that